src/theory/builtin/theory_builtin_type_rules.h

```diff
@@ -192,6 +192,41 @@ class LambdaTypeRule {
   }
 };/* class LambdaTypeRule */
 
+class ChoiceTypeRule
+{
+ public:
+  inline static TypeNode computeType(NodeManager* nodeManager,
+                                     TNode n,
+                                     bool check)
+  {
+    if (n[0].getType(check) != nodeManager->boundVarListType())
+    {
+      std::stringstream ss;
+      ss << "expected a bound var list for CHOICE expression, got `"
```

Comment:
I assume that char ` should be ' in this case?

Reply:
Actually I was following the convention in other error messages (e.g. line 152 of this file).

Reply:
OK, as long as it is intentional ;-)